### PR TITLE
Improved description for envelope property for S3/GCS connectors.

### PR DESCRIPTION
**Pull request by tomasz-sadura**

### CODE REVIEW

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcs-connector.adoc [hunk 1]
```diff
@@ -116,6 +116,10 @@ a| The template for file names on GCS. Supports `{{ variable }}` placeholders fo
 | `format.output.fields.value.encoding`
 | The type of encoding to be used for the value field. Supported values are: 'none' and 'base64'.
 
+| `Envelope for primitives`
+| `format.output.envelope`
+| Whether to enable additional JSON object wrapping the actual value.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcs-connector.adoc]:
```suggestion
| Specifies whether or not to enable additional JSON object wrapping of the actual value.
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-s3-sink-connector.adoc [hunk 2]
```diff
@@ -143,7 +143,7 @@ a| The template for file names on S3. Supports `{{ variable }}` placeholders for
 
 | `Envelope for primitives`
 | `format.output.envelope`
-| Whether to enable envelope for entries with single field.
+| Whether to enable additional JSON object wrapping the actual value.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-s3-sink-connector.adoc]: Specifies whether or not to enable additional JSON object wrapping of the actual value.

Comment (Feediver1) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/create-s3-sink-connector.adoc]: Let me know if this is okay--if so, I can commit and merge @tomasz-sadura

Comment (tomasz-sadura) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/create-s3-sink-connector.adoc]: Yes, sounds better, please feel free to commit and merge.

Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-s3-sink-connector.adoc:146]: Specifies whether or not to enable additional JSON object wrapping of the actual value.

Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-s3-sink-connector.adoc]:
```suggestion
| Specifies whether or not to enable additional JSON object wrapping of the actual value.
```

**Review by Feediver1 — APPROVED** [review]

**Pull request merged**